src/components/FilesList.js

```diff
@@ -20,7 +20,15 @@ const columns = [
   }
 ];
 
+const dropMenu = (
+  <Menu>
+    <Menu.Item key="1">Delete</Menu.Item>
+    <Menu.Item key="2">Host Preference</Menu.Item>
+  </Menu>
+);
+
 const components = onMove => ({
+
```

Comment:
I think for this file we can wrap the `tr`, just like we do the `div` in `FilesGrid`.